### ISSUE TITLE
FileSystem.CopyFile Success or Fail

### ISSUE THREAD

**Issue by scubaman120** [DynamoDS/Dynamo#11393]
Dynamo version

2.3.0.5885

Operating system.

Windows 10

## What did you do? 

File System.CopyFile will output null regardless if it works or fails. 

![image](https://user-images.githubusercontent.com/33437179/104063944-ecde8780-51b1-11eb-9461-bc24cdc21901.png)

## What did you expect to see?

It would be nice if it exported true for files that were successfully copied and false for files that were not copied. 

![image](https://user-images.githubusercontent.com/33437179/104064060-139cbe00-51b2-11eb-8ded-6683fe705f1f.png)

```
import shutil

original = IN[0]
target = IN[1]

List = [] 
for o, t in zip(original,target):
	try:
		shutil.copyfile(o, t)
		List.append(True)
	except:
		List.append(False)
		
OUT = List
```

## What did you see instead?

null

**Comment by QilongTang** [comment]
Tracked internally at https://jira.autodesk.com/browse/DYN-3454

**Comment by mjkkirschner** [comment]
There is another task about updating all nodes that return "void" with better results - seems these can be merged.

**Comment by QilongTang** [comment]
This issue will be fixed in the upcoming Dynamo 2.12.0 release. See linked pull request for screenshot of improvements. Closing and thanks for reporting!
